### PR TITLE
View results flow change

### PR DESCRIPTION
**Pull request by Dheeraj-DJP**
View results page modifications:
1. Download report and results buttons
2. Fix the decimal places of ct value to 2
3. For the new JSON info where ct value is an array, print only second element (mean ct value)

Reupload correct:
1. Buttons for reupload for Tapestry results pending and error in parsing
2. Sweet alerts for the above to help user understand the issues with reuplaoding

In addition, this PR also corrects an error in Test component where useEffect for api call on page load was used wrongly resulting in infinite loop.

### CODE REVIEW

File: src/views/LandingPage/FirstTime/index.js [hunk 1]
```diff
@@ -76,14 +72,23 @@ function FirstTime() {
...
-    const count = pageInfo ? pageInfo.pagination.count : 0;
-    if (count !== 0) {
-      history.push("/ongoing-tests");
+    if (pageInfo !== undefined) {
+      const count = pageInfo ? pageInfo.pagination.count : 0;
+      if (
+        count !== 0 &&
+        count !== "0" &&
+        count !== null &&
+        count !== undefined
```
Comment (vyasakanksha) [inline src/views/LandingPage/FirstTime/index.js:81]: what is the type of `count`. You only need one check, not 4

File: src/views/LandingPage/FirstTime/index.js [hunk 2]
```diff
@@ -107,11 +112,13 @@ function FirstTime() {
     if (!initialMount) {
       setLoading(false);
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [labUsersList]);
 
   return (
     <LoadingOverlay active={loading} spinner text="Loading configuration">
       <div className="first-time">
+        {console.log(pageInfo)}
```
Comment (vyasakanksha) [inline src/views/LandingPage/FirstTime/index.js:121]: remove this

Comment (Dheeraj-DJP) [reply src/views/LandingPage/FirstTime/index.js:121]: Yeah, sure.

**Comment by vyasakanksha** [comment]
approved

**Pull request merged**